modules/administration-guide/pages/configuring-the-open-vsx-registry-url.adoc

```diff
@@ -22,6 +22,11 @@ spec:
        openVSXURL: __<your_open_vsx_registy>__
 # [...]
 ----
++
+[WARNING]
+====
+link:https://marketplace.visualstudio.com/vscode[Visual Studio Code Marketplace] is not supported by the {prod} editors running in the browser due to the dedicated Microsoft’s link:https://cdn.vsassets.io/v/M190_20210811.1/_content/Microsoft-Visual-Studio-Marketplace-Terms-of-Use.pdf[Terms of Use].
```

Comment:
```suggestion
Due to the dedicated Microsoft link:https://cdn.vsassets.io/v/M190_20210811.1/_content/Microsoft-Visual-Studio-Marketplace-Terms-of-Use.pdf[Terms of Use], extensions from link:https://marketplace.visualstudio.com/vscode[Visual Studio Code Marketplace] are not supported by {prod} editors that run in a browser.
```

Reply:
@deerskindoll I believe it is a similar situation as with the previous PR - https://github.com/eclipse-che/che-docs/pull/2832
Please, read the whole section, the focus should be on the marketplace/registry, not individual extensions

Reply:
I see what you mean. I think my edit still works because the doc refers to extensions from open vsx but how about this:

Due to the dedicated Microsoft link:https://cdn.vsassets.io/v/M190_20210811.1/_content/Microsoft-Visual-Studio-Marketplace-Terms-of-Use.pdf[Terms of Use], link:https://marketplace.visualstudio.com/vscode[Visual Studio Code Marketplace] is not supported by {prod}.

Reply:
this one is good ^

Reply:
@deerskindoll could you please drop `extensions from` from the last suggestion and I will apply